README.md

```diff
@@ -246,7 +196,7 @@ On a disconnect, the `status` argument is set to `REDIS_OK` when disconnection w
 user, or `REDIS_ERR` when the disconnection was caused by an error. When it is `REDIS_ERR`, the `err`
 field in the context can be accessed to find out the cause of the error.
 
-You not need to reconnect in the disconnect callback, hiredis-vip will reconnect this connection itself
+You not need to reconnect in the disconnect callback, hiredis-cluster will reconnect this connection itself
```

Comment:
"reconnect <del>this connection</del> by itself" => "reconnect by itself"

Also add a period (.) in the end.

Reply:
done